general/press.rst

```diff
@@ -191,8 +191,8 @@ Logos and Graphic material
 .. raw:: html
 
   <div align="center" class="align-center">
-    <img alt="../_downloads/openwisp-logo.svg"
-         src="../_downloads/openwisp-logo.svg"
+    <img alt="../../assets/design/openwisp-logo.svg"
+         src="../../assets/design/openwisp-logo.svg"
```

Comment:
if I delete the build and static assets dir and recreate it, I can't see the logo.
Can I know why are you changing this bit? I don't understand what problem was causing and why we need to change it.

Reply:
I change it because the older link was not working and pylink was showing that link as broken.

Reply:
Thanks for explaining. If you delete the build and static directories, then recreate the documentation with `make html`, can you see the logo? I see this:

![Screenshot from 2020-10-04 16-09-55](https://user-images.githubusercontent.com/841044/95027171-2015f580-065c-11eb-8576-7a71312ad9ed.png)

Reply:
PS: the link works though, it opens the right logo. It's the src attribute of the image which does not seem to work.

Reply:
It was error with path. It seems to work in starting when I tried but don't know why it fails, although corrected it now. Else shouldn't we use the path as `https://github.com/openwisp/openwisp2-docs/blob/master/assets/design/openwisp-logo.svg`.